tests/runner/Testcontracts.ml

```diff
@@ -292,6 +292,8 @@ let build_misc_tests env =
       [
         "-init";
         tests_dir_file env.tests_dir test_ctxt ("init_bad" ^ snum ^. "json");
+        "-gaslimit";
+        testsuit_gas_limit;
```

Comment:
This looks odd. Don't we still need to supply a gas limit to scilla-runner, even if we move the size charge out of scilla-runner?

Reply:
> This looks odd. Don't we still need to supply a gas limit to scilla-runner, even if we move the size charge out of scilla-runner?

We do. This particular line is a fix to a long standing hidden bug where we expect a failure, but it was failing all this while because we weren't providing the gas limit at all !. Now it'll fail for the right reasons.

Reply:
Oh. Our test system is a bit fragile, isn't it? :-)